sass/variables/variables.scss

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
the name of the file should be `_variables`,  otherwise, compiler creates variables folder with empty file in your css/ folder